.github/actions/migrate/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Migrate Nx Workspace
+description: Migrate Nx Workspace to the specified version
+inputs:
+  nx-version:
+    description: Nx version to try to migrate
```

Comment:
```suggestion
    description: Nx version to try to migrate
    default: 'next'
```

Reply:
I have plans to reuse this to automate the Nx Workspace migration. I don't want to couple this action with `next` for now.